src/ConstructiveSolidGeometry/PointsAndVectors/Vectors.jl

```diff
@@ -0,0 +1,53 @@
+
+struct PlanarVector{T} <: AbstractPlanarVector{T}
+    u::T
+    v::T
+end
```

Comment:
Are `PlanarVectors` used?

Reply:
Not in the current used (`include("<fn>")`)-CSG Files.

Reply:
I will leave it for now. At the very end when all primitives are implemented we can do a clean up.